test/function_tests/backplane_operator_install_test/backplane_operator_install_test.go

```diff
@@ -678,3 +727,13 @@ func validateResourceSpecs(namespace string, nodeSelector map[string]string, pul
 
 	}, time.Minute, time.Second).Should(Succeed())
 }
+
+func contains(s []string, str string) bool {
```

Comment:
We have this func here - https://github.com/redhat-openshift-ecosystem/community-operators-prod/tree/main/operators/assisted-service-operator/0.2.12
Can we reuse? It might cause import cycle...

just a suggestion, not necessary for approval

Reply:
I thought it better to keep the helper function contained within the same package because it's so basic and would allow the tests to not be dependent on the importing the main code.